.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,24 +16,9 @@ The status of your PR. ie WIP or Finished.
 Ex: Finished; Pending Review 

```

Comment:
remove the comments, use a checklist (you can add a comment on how to check a box)
```
- [ ] WIP
- [ ] Pending Review
- [ ] Finished
```